modules/kubernetes/pages/k8s-operator/backup-and-restore.adoc

```diff
@@ -0,0 +1,710 @@
+Tag:                      daily
+Timeout:                  18000
+Cluster Name:               test-cluster
+Destination:
+Local:
+Path:   /home/tigergraph/backup
+Storage:  local
+Status:
+Conditions:
+Last Transition Time:  2022-12-16T13:44:24Z
+Message:               Failed to backup cluster
+Reason:                BackupFailed
+Status:                True
+Type:                  Failed
+Start Time:              2022-12-16T13:44:03Z
+Target Ready:            true
+Events:
+Type     Reason                Age                   From              Message
+----     ------                ----                  ----              -------
+Normal   Target cluster ready  31m (x35 over 3d12h)  TigerGraphBackup  Target cluster is ready for backup
+Warning  Backup job failed     31m (x12 over 3d12h)  TigerGraphBackup  Failed to backup cluster test-cluster
+You can see the Events that Backup job failed, which means this backup job is failed.
+----
+
+== Schedule backups
+
+=== Create and manage a backup schedule
+
+[source.wrap, console]
+----
+USAGE:
+kubectl tg backup-schedule [create|update|list|pause|resume] [OPTIONS]
+Options:
+-h|--help: show this message
+-n|--namespace :              set namespace to deploy TG cluster, default namespace is current namespace
+-c|--cluster-name :           (required)set cluster-name to deploy TG cluster, no default
+--tag :                       (required)specify the tag of backup files. e.g. if you specify --tag daily, the backup file will be daily-20xx-xx-xxTxxxxxx
+--staging-path :              specify where to store the temporary files
+--timeout :                   the backup timeout in seconds,default: 18000
+--compress-process-number :   the number of concurrent process for compression during backup
+value 0 means the number of processes used to compress equals
+the number of the node's CPU cores. And the default value is 0
+--schedule :                  specify the schedule of backup in cron format. e.g. '* * * * *' is backup every minute
+--incremental :               do incremental backup
+--full :                      do full backup (full backup is performed by default)
+--max-retry :                 set max times of retry for each backup
+--max-backup-file :           set the max number of files you want to retain
+--max-reserved-day :          set the max number of days you want to retain these backups
+--destination :               set the destination to store backup files, support local and s3 now
+Followings are about the configuration of different destination:
+If destination is local,you should provide:
+--local-path :              set the local path where to store backup files
+If destination is s3:
+--s3-bucket :               S3 Bucket name
+--aws-access-key-id :       the aws access key id
+--aws-secret-access-key :   the aws secret access key
+----
+
+Just like backup, a cluster and a tag correspond to a backup-schedule configuration, which is called a CR (custom resource) in Kubernetes.
```

Comment:
This sentence is confusing me. I don't understand the use of "correspond" here.
I think the problem is the verb tense: the present tense, with a very strong existential sense.
I think the more appropriate tense is conditional:
"A cluster and tag can have a corresponding backup-schedule configuration."  Correct?  Because not every cluster and tag will necessarily have a backup schedule.